### ISSUE TITLE
Cannot represent as object a 'Contributor' with Role 'Other'

### ISSUE THREAD

**Issue by c-martinez** [ITC-CRIB/fairly#38]
I'm trying _fairly_, just to get familiar with the tool. I tried to use it download some files from Zenodo, and ran into the following error:

```bash
ruamel.yaml.representer.RepresenterError: cannot represent an object: {..., 'role': 'Other'}
```
I think it due to the metadata in the Zenodo record not being handled properly, specifically it seems like contributors are causing the issue.

The following lines should be sufficient to reproduce the error:
```python
import fairly

doi = "https://doi.org/10.5281/zenodo.1494140"
downloadDir = 'download/'

dataset = fairly.dataset(doi)
local_dataset = dataset.store(downloadDir)
```

**Comment by manuGil** [comment]
Could you tell me if you install *fairly* from PyPI or from source?

**Comment by manuGil** [comment]
Hello Carlos, 
I tried to reproduce the error. But I don't see it. I tried the PyPI version and the source version. **Does the error appear at the beginning of the download or in the middle?** In my attempt to reproduce the error, I see the manifest file is written, and the data start downloading. 

Can you confirm that the error appears for other Zenodo datasets?

**Comment by c-martinez** [comment]
Hello @manuGil !

The error appears at the beginning of the download (as far as I can tell). A manifest file is created, but it is empty. I tried with a [different dataset](https://doi.org/10.5281/zenodo.1189327), but that one worked.

I tried on a different machine, and it worked there, so I tracked the bug down, it seems to be that the difference is in versions of python and ruamel.yaml versions:

- Python 3.9, ruamel.yaml==0.17.26 ==> worked
- Python 3.10.8, ruamel.yaml==0.17.21 ==> failed

So probably it is an issue with the version of ruamel.yaml?
